Pokestop Shows Coords

l3uddz opened the pull request:
Coords are now displayed for each pokestop in the log messages.

nicoschmitt commented:
It's call "Latitude" and "Longitude" all over the app, you should stick with the same name

l3uddz commented:
changed.
also added PokemonsToPokeball option so that you can essentially limit specific pokemon to only regular pokeballs.
also tweaked the GetBestBall func, do try it out, seems to be working better, although would be cool to maybe remember a high IV/CP pokemon when out of balls, go farm some, and come back if within time limit and catch it.
Also made it so that the first attempt is always a pokeball (if available), we can call it the lucky ball, sometimes it catches what would have typically been a great ball/ultra ball, e.g:
[22:37:45] (INFO) [Sniper] Sniping a Charizard with unknown IV at -33.8618424569,151.080662609...
[22:37:49] (PKMN) (CatchSuccess) | (Normal) Charizard Lvl: 18 CP: (1195/1338) IV: 25.00% | Chance: 3.53% | 16999325.94m dist | with a PokeBall (5 left). | Candies: 84
---
[22:38:03] (INFO) [Sniper] Sniping a Lapras with 67 IV at 42.462260895562,-82.961959687368...
[22:38:06] (PKMN) (CatchSuccess) | (Normal) Lapras Lvl: 27 CP: (2193/2299) IV: 66.00% | Chance: 11.52% | 6025097.22m dist | with a PokeBall (3 left). | Candies: 388

DurtyFree commented:
Please do seperate PRs for the different additions @l3uddz 

Will close this, submit new PRs then
